src/components/homepage-hero/style.module.css

```diff
@@ -22,16 +22,14 @@
 }
 
 .heading {
-  composes: ds-token-typography-heading-0 from global;
   color: var(--token-color-palette-neutral-700);
-  margin: 0;
-  padding-bottom: 8px;
+  margin-bottom: 8px;
```

Comment:
Swapped `padding-bottom` for `margin-bottom` here because it's should be the spacing outside of the component using this class.